sdk/go/common/workspace/plugins.go

```diff
@@ -587,7 +587,7 @@ func GetPluginPath(kind PluginKind, name string, version *semver.Version) (strin
 
 	// If we have a version of the plugin on its $PATH, use it, unless we have opted out of this behavior explicitly.
 	// This supports development scenarios.
-	if os.Getenv("PULUMI_IGNORE_AMBIENT_PLUGINS") != "" {
+	if _, isFound := os.LookupEnv("PULUMI_IGNORE_AMBIENT_PLUGINS"); !isFound {
```

Comment:
It seems you're inverting the logic. Seems to be the same as:

```
os.Getenv("PULUMI_IGNORE_AMBIENT_PLUGINS") == ""
```

But more emphatically readable !isFound.

Judging by the comment, the correction is needed here yes. The IGNORE env var should skip the logic.

Reply:
yeah that's exactly it - we should skip loading from PATH if that env var is set